src/plugins/intel_gna/src/gna_device.hpp

```diff
@@ -47,6 +47,7 @@ class GNADeviceHelper : public GNADevice {
     uint32_t nGnaDeviceIndex = 0;
     bool useDeviceEmbeddedExport = false;
     uint32_t maxLayersCount_ = 0;
+    size_t m_mem_alignment = 0;
```

Comment:
1. why not to set this value in header?
```
size_t m_mem_alignment  {limitations::kMemoryAlignment};
```
2. I don't see place where this value is changes shouldn't it be const/constexpr ?